### PR TITLE
Draft version of projector.toggle_stable action. Update tests.

### PR DESCRIPTION
**Pull request by reiterl**
See the questions in the issue.

### CODE REVIEW

File: openslides_backend/action/actions/projector/toggle_stable.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    def get_updated_instances(self, action_data: ActionData) -> ActionData:
+        for instance in action_data:
+            for projector_id in instance["ids"]:
+                filter_ = And(
+                    FilterOperator("current_projector_id", "=", projector_id),
+                    FilterOperator(
+                        "content_object_id", "=", instance["content_object_id"]
+                    ),
+                    FilterOperator("type", "=", instance.get("type")),
```
Comment (jsangmeister) [inline openslides_backend/action/actions/projector/toggle_stable.py:60]: If `type` is not set in the payload, you mustn't add this filter

File: openslides_backend/action/actions/projector/toggle_stable.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                    FilterOperator("stable", "=", True),
+                )
+                results = self.datastore.filter(
+                    Collection("projection"), filter_, ["id"]
+                )
+                if results:
+                    for id_ in results:
+                        self.execute_other_action(ProjectionDelete, [{"id": id_}])
```
Comment (jsangmeister) [inline openslides_backend/action/actions/projector/toggle_stable.py:68]: Please reduce this to one action call by giving all ids in the action data.

**Comment by reiterl** [comment]
Updated the payload to include the `meeting_id` and simplify the code.

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/projector/toggle_stable.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/projector/toggle_stable.py]: Okay, add the type filter only if type is set.

File: openslides_backend/action/actions/projector/toggle_stable.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/projector/toggle_stable.py]: Changed the code here, the ProjectorDelete action is now called once.

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**